.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,15 @@
 
 **What this PR does / why we need it**:
 
-**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
 
 **Special notes for your reviewer**:
 
 **Release note**:
-<!--  Steps to write your release note:
-1. Use the release-note-* labels to set the release note state (if you have access)
-2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`.
+<!--  Write your release note:
+Enter your extended release note in the below block. If no release note is required, just write `NONE` (case insensitive, don't include quotation marks).
```

Comment:
Should we mention the effect of "action required" or is that overly verbose for the PR template?
Should we link to the release note process documentation that you updated in the community repo?

Reply:
>Should we mention the effect of "action required" or is that overly verbose for the PR template?

makes sense to tell a bit about how to write "action required" release note.

>Should we link to the release note process documentation that you updated in the community repo?

this is already in the template in L4: https://github.com/kubernetes/kubernetes/pull/54114/files#diff-195a635ad245ded9076330e31134bd80R4